qa/tasks/tests/test_cephadm.py

```diff
@@ -1,70 +0,0 @@
-from tasks import cephadm
```

Comment:
@sebastian-philipp i don't understand how this is related to the renaming. could you elaborate it a little bit?

Reply:
should be in a different commit, indeed. Thanks